packages/runtime/test-runtime-utils/package.json

```diff
@@ -23,7 +23,7 @@
     "lint:fix": "npm run eslint:fix",
     "test": "npm run test:mocha",
     "test:coverage": "nyc npm test -- --reporter mocha-junit-reporter --reporter-options mochaFile=nyc/junit-report.xml",
-    "test:mocha": "mocha --recursive dist/test -r node_modules/@fluidframework/mocha-test-setup --unhandled-rejections=strict",
+    "test:mocha": "mocha --inspect-brk --recursive dist/test -r node_modules/@fluidframework/mocha-test-setup --unhandled-rejections=strict",
```

Comment:
Remember to revert, right? :)

Reply:
the pr build will never pass with this here